gpt_computer_agent/__init__.py

```
@@ -2,6 +1,288 @@
+try:
+    from .start import start
```

Ruff (0.8.2)

`.start.start` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)